cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -51,7 +51,7 @@
 #  - Document functions using proper syntax:
 #    https://technet.microsoft.com/en-us/library/hh847834(v=wps.620).aspx
 
-$INFRA_CONTAINER = 'gcr.io/gke-release/pause-win:1.1.0'
+$INFRA_CONTAINER = 'k8s.gcr.io/pause:3.4.1'
```

Comment:
please let's add this to build/dependencies.yaml in the "dependents" entry so we don't de-sync it again 😅

Reply:
sorry, did not follow you, add where? should we remove it from this file? I'm sorry for the dumb question, not clear to me

Reply:
not at all, sorry somehow I thought I'd already seen you interact with this:
https://github.com/kubernetes/kubernetes/blob/473af0b8d1951b5c362df60925bb8d9fb007fab9/build/dependencies.yaml#L162

we have a presubmit that ensures that the versions are in sync based on the (filepath, regex) specs in that file.

Reply:
we should add these references to the pause image to that list so when pause is updated again someday the presubmit will catch if these new references added in this PR aren't updated

Reply:
LOL wrong repo. bentheelder is taking a long nap soon.

Reply:
wishful thinking. sorry about that!

Reply:
🙃